k8_cortx_cloud/solution.yaml

```diff
@@ -21,6 +21,15 @@ solution:
     zookeeper: 3.7.0-debian-10-r182
     rancher: rancher/local-path-provisioner:v0.0.20
   common:
+    metallb:
```

Comment:
We should add in an `enabled` field in here to use for the deployMetalLb function in deploy-cortx-cloud.sh , as that will allow us to leave the block of sensible defaults here but a default `enabled` value of `false`.